src/app/VerifyActivities/components/VerifyActivitiesContainer.js

```diff
@@ -42,16 +44,21 @@ export class VerifyActivitiesContainer extends Component {
   filterActivitiesByInReviewStatus = activities => (
     activities.filter(activity => activity.status === ACTIVITY_STATUS.IN_REVIEW))
 
+  handleVerify = (loggedActivityId, status) => {
+    const { verifyActivity } = this.props;
+    verifyActivity(loggedActivityId, status);
+  }
+
   render() {
     const { society, societyName } = this.props;
     let verifyActivitiesHtml = (<LoaderComponent className='loader' />);
     if (societyName) {
+      const { inReview } = society;
```

Comment:
This state is not necessary. There is a function, `filterActivitiesByInReviewStatus`, that was doing the same thing but you removed it. Is there a reason why?

Reply:
@Chris-Maina I decided to have this in the redux store so that it would be easier to update the list when an item is rejected or approved